certora/confs/ReentrancyEthereum.conf

```diff
@@ -0,0 +1,19 @@
+{
+  "files": [
+    "src/MorphoTokenEthereum.sol",
+  ],
+  "parametric_contracts" : ["MorphoTokenEthereum"],
+  "solc": "solc-0.8.27",
+  "verify":"MorphoTokenEthereum:certora/specs/Reentrancy.spec",
+  "rule_sanity": "basic",
+  "server": "production",
+  "packages": [
+        "@openzeppelin/contracts/=lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts"
+  ],
+  "prover_args": [
+        "-enableStorageSplitting false"
+    ],
```

Comment:
I can't remember what this does and why we need it :sweat_smile:

Reply:
Removed, I added it in the configuration because verif was failing due to this but it doesn't seem necessary anymore. I'll try to find the run and see what was it exactly.